Проектная работа №1

KateSilencio opened the pull request:
Калькулятор счета
1й прототип программы

KateSilencio committed:
Проектная работа №1
Калькулятор счета
1й прототип программы

doc-ODA reviewed (commented):
Привет! Ты молодец, почти все с первого раза сделала!

doc-ODA commented on src/main/java/Main.java:
```diff
@@ -1,6 +1,171 @@
+            while (true){
+                System.out.println("Введите название товара: ");
+                newName = scanner.next();
+                boolean flPutProduct = false; //флаг уже положенного товара
```
хм, тут я логически не совсем согласен, товары же могут быть с похожими названиями? ну или тогда перезаписывать цену? или складывать ? но это так, порассуждать, за крит не считаю, так что финальное решение на твое усмотрение.

KateSilencio committed:
Проектная работа №1
Калькулятор счета
2я версия программы
Внесенные изменения:
1. дополнительные проверки в enterPerson
2 в createProduct:
2.1 формат ввода названия
2.2 проверка на пробелы и пустую строку
2.3 добавлена цена в сообщение вывода одинакового товара
2.4 формат ввода команды Завершить
3.дополнительные проверки в getPrice
4. в calculateAndPrint добавлено форматирование вывода общей цены sum
